NAMESPACE

```diff
@@ -1,16 +1,16 @@
 # Generated by roxygen2: do not edit by hand
 
-export(add_obs_error_to_nowcast)
-export(add_obs_errors_to_nowcast)
+export(add_obs_err_to_pt_nowcast_mat)
+export(add_obs_errs_to_pt_nowcast_mat)
 export(aggregate_df_by_ref_time)
 export(apply_delay)
-export(convert_reporting_square_to_df)
+export(convert_nowcast_matrix_to_df)
 export(estimate_dispersion)
-export(generate_point_nowcast)
-export(generate_point_nowcasts)
+export(generate_pt_nowcast_mat)
+export(generate_pt_nowcast_mat_list)
 export(generate_triangles)
 export(get_delay_estimate)
-export(nowcast_list_to_df)
+export(nowcast_matrix_list_to_df)
 export(replace_lower_right_with_NA)
 export(truncate_triangle)
 export(truncate_triangles)
```

Comment:
is this becase the output is a list as is the input? Shoud it use a list prefix then? 

As above could be as_triangle_list?

Reply:
I prefer to over as, it implies more action to me as to what is happening 

this takes a list and converts to df?

Reply:
Oh are you referring to truncate triangles here 
This takes a single reporting triangle and returns a list of truncated reporting triangles, but the key thing its doing is truncating the original triangle

Reply:
No -- it is taking a matrix and generating a list of `n` matrices, which have one row sliced off of the original matrix.